.github/workflows/virus-scan.yml

```diff
@@ -6,92 +6,12 @@ jobs:
   virus-scan:
```

Comment:
This repo has .reposync so I am not sure why this file didn't get updated with recent changes, but I just copied them over manually.

Reply:
It did, but your PR started earlier than that change. But if they're the same changes they should merge in just fine.